st2common/st2common/content/bootstrap.py

```diff
@@ -56,10 +56,15 @@ def register_opts():
         cfg.BoolOpt('policies', default=False, help='Register policies.'),
         cfg.BoolOpt('configs', default=False, help='Register and load pack configs.'),
 
+        # Config related options
+        cfg.BoolOpt('validate-configs', default=False,
```

Comment:
I've no idea why we want a flag for this? If there is a schema, we should by default validate and throw an error. We now have flags for skip failure or abort (default). I think those should be enough.